lib/python/qmk/keymap.py

```diff
@@ -18,6 +18,19 @@
 const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
 __KEYMAP_GOES_HERE__
 };
+
+void encoder_update_user(uint8_t index, bool clockwise) {
+__ENCODERS_GO_HERE__
+};
+"""
+
+ENCODER_IF = """\t_ELIF_ (index == _INDEX_) {
+\t\tif (clockwise) {
+\t\t\ttap_code16(_CLOCKWISE_);
+\t\t} else {
+\t\t\ttap_code16(_COUNTER_);
+\t\t}
+\t}
 """
```

Comment:
Just a little, seemingly unnecessary change here:

```suggestion
\t}"""
```